src/Tools/Directory.Build.targets

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.targets"/>
+  <!--
+    Target that adds CustomPackageFile files to the nupkg at the path specified by the PackagePath item metadata.
+  -->
+  <Target Name="GatherCustomPackageFiles" DependsOnTargets="$(GatherCustomPackageFilesDependsOn)">
```

Comment:
Maybe a way to not depends on the internal is something like: https://github.com/dotnet/diagnostics/blob/4c0a75d8c11618c733efd7a87b2744a9ba4e805f/src/dbgshim/pkg/Microsoft.Diagnostics.DbgShim.props#L25-L31

Reply:
Sure, I'll take a look at that.

Reply:
Unfortunately, that particular property doesn't work because of:
- https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L374
- https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L460

due to IncludeBuildOutput being forced to false due to https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets#L46 since this is a tool package whereas the one that you referenced is a library package.

Reply:
I was able to do it with the TargetsForTfmSpecificContentInPackage property.